js/directives/day-block.js

```diff
@@ -15,7 +15,7 @@
 			'</div>'+
 
 			'<div ng-style="controller.monthStyleOverride()" ' +
-			'class="month">{{controller.date | luifMoment: \'MMM\' | limitTo : 3}}'+
+			'class="month">{{controller.date | luifMoment: \'MMM\' | fancyMonth}}'+
```

Comment:
why not just keep `controller.date | luifMoment: \'MMM\`, it'll use the monthshort defined in the moment's [locale](http://momentjs.com/docs/#/i18n/)